modules/account/src/main/kotlin/edu/stanford/spezi/module/account/register/RegisterValidation.kt

```diff
@@ -0,0 +1,29 @@
+package edu.stanford.spezi.module.account.register
+
+import java.time.LocalDate
+
+fun isEmailValid(email: String): Boolean =
```

Comment:
Instead of global functions, these should be handled in a `class RegistratationFormValidator` that you can inject in the view model